tests/testsuite.src/run_misc.at

```diff
@@ -15251,3 +15251,324 @@ AT_CHECK([$COMPILE prog.cob])
 AT_CHECK([$COBCRUN_DIRECT ./prog], [0], [1.2345E-5 1.2345E-5], [])
 
 AT_CLEANUP
+
+
+AT_SETUP([profiling and codegen])
+AT_KEYWORDS([EVALUATE])
+
+AT_DATA([prog.cob], [
+IDENTIFICATION DIVISION.
+PROGRAM-ID. PROG.
+
+DATA DIVISION.
+WORKING-STORAGE SECTION.
+    01 STR PIC X(6) VALUE "..-..-".
+    01 OFFSET BINARY-LONG UNSIGNED.
+
+PROCEDURE DIVISION.
+    MOVE 1 TO OFFSET
+    PERFORM UNTIL OFFSET > LENGTH OF STR
+        CALL "TestRecurse" USING STR OFFSET
```

Comment:
I _guess_ one recursive test is enough, if this test should also have it then move the "simple recursive call" up front